packages/core/src/base-axis-chart.ts

```diff
@@ -244,6 +245,15 @@ export class BaseAxisChart extends BaseChart {
 				.text(this.options.scales.x.title);
 		}
 
+		// Add chart title
+		if (this.options.scales.headers.title) {
```

Comment:
Adding the chart title is not relevant to `setXAxis`. Should be moved into a different function/area.